js/index.js

```diff
@@ -40,3 +40,60 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let navContent = document.getElementsByTagName("a");
+navContent[0].textContent = siteContent.nav["nav-item-1"];
+navContent[1].textContent = siteContent.nav["nav-item-2"];
+navContent[2].textContent = siteContent.nav["nav-item-3"];
+navContent[3].textContent = siteContent.nav["nav-item-4"];
+navContent[4].textContent = siteContent.nav["nav-item-5"];
+navContent[5].textContent = siteContent.nav["nav-item-6"];
+
+let ctaImgID = document.getElementById("cta-img")
+//console.log(ctaImgID)
+ctaImgID.src = "img/header-img.png";
+
+let ctaText = document.getElementsByClassName("cta-text")
+
+//console.log(ctaText)
+
+ctaText[0].children[0].innerHTML = siteContent.cta["h1"];
+ctaText[0].children[1].textContent = siteContent.cta["button"];
+
+let textContentStuff = document.querySelectorAll(".text-content")
```

Comment:
Remember, any valid CSS selector works for the query selectors. That means you can do something like:

```js
let topTextContentStuff = document.querySelectorAll(".top-content .text-content");
```

This will target the two `.text-content` divs which are children of `.top-content`. This allows you to be very specific in your selections without needing to modifying the original HTML with additional classes or ID's.